plugins/out_es/es.c

```diff
@@ -1052,6 +1081,11 @@ static struct flb_config_map config_map[] = {
      "When enabled, generate _id for outgoing records. This prevents duplicate "
      "records when retrying ES"
     },
+    {
+     FLB_CONFIG_MAP_STR, "write_operation", "index",
```

Comment:
Currently, we use "create" op  as a default.
https://github.com/fluent/fluent-bit/blob/v1.8.7/plugins/out_es/es_bulk.h#L28

I think it is better to use "create" as a default to prevent breaking change.